### PR TITLE
Updated company Iguazio 515156685

### PR DESCRIPTION
**Pull request by ZeevRispler**

### CODE REVIEW

File: companies/515156685.json [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+  "name": "Iguazio (acquired by McKinsey)",
+  "description": "A platform that enables development, deployment and management of AI and Gen AI projects at scale to generate real business impact in the enterprise.",
   "categoryId": 1,
   "size": "m",
   "logoFile": "logos/iguazio_systems-logo.webp",
   "websiteUrl": "https://www.iguazio.com/",
   "careersUrl": "https://www.iguazio.com/careers/",
   "isHiring": true,
-  "linkedinId": null,
+  "linkedinId": "https://www.linkedin.com/company/iguazio/",
```
Comment (mluggy) [inline companies/515156685.json:10]: linkedinId only (i.e iguazio)

File: companies/515156685.json [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
   "crunchbaseId": "iguaz-io",
   "finderId": "iguaz-io",
   "xId": null,
   "addresses": [
     {
       "street": "שנקר אריה",
-      "houseNumber": 5,
+      "houseNumber": 18,
       "city": "הרצליה",
       "lat": 32.1596535,
```
Comment (mluggy) [inline companies/515156685.json:19]: if 18 (instead of 5) street changes the lat/lon either keep them blank or update as well.

**Review by mluggy — CHANGES_REQUESTED** [review]

File: companies/515156685.json (hunk 1 above)
Comment (ZeevRispler) [reply companies/515156685.json:10]: Done

File: companies/515156685.json (hunk 2 above)
Comment (ZeevRispler) [reply companies/515156685.json:19]: they were correct (checked in goolgl maps), but the addres was not (idk how)

**Pull request merged**